protocol/query/method/message/data/dataFederationResult.json

```diff
@@ -0,0 +1,70 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "$id": "https://raw.githubusercontent.com/dedis/popstellar/master/protocol/query/method/message/data/dataFederationResult.json",
+    "description": "Sent by an server to a remote server, to inform them about the result of the authentication procedure",
+    "type": "object",
+    "oneOf": [
+        {
+            "type": "object",
+            "properties": {
+                "object": {
+                    "const": "federation"
+                },
+                "action": {
+                    "const": "result"
+                },
+                "status": {
+                    "type": "string",
+                    "$comment": "status of the authentication attempt"
+                },
+                "reason": {
+                    "type": "string",
+                    "$comment": "to be used in failures, describing the error that happened"
+                },
+                "challenge": {
+                    "$ref": "../message.json",
+                    "$comment": "message/message containing a FederationChallenge data"
+                }
+            },
+            "additionalProperties": false,
+            "required": [
+                "object",
+                "action",
+                "status",
+                "challenge"
+            ]
+        },
+        {
+            "type": "object",
+            "properties": {
+                "object": {
+                    "const": "federation"
+                },
+                "action": {
+                    "const": "result"
+                },
+                "status": {
+                    "type": "string",
+                    "$comment": "status of the authentication attempt"
+                },
```

Comment:
The status value could be restricted to be only the specific value "success", either with pattern or with enum or with const.